src/pages/pricing/index.tsx

```diff
@@ -34,125 +36,74 @@ const Pricing: FunctionComponent<PricingProps> = ({redirectURL}) => {
     const {annualPrice} = prices
     const account = first<StripeAccount>(get(viewer, 'accounts'))
     await track('checkout: selected plan', {
-      priceId: annualPrice.price_id,
+      priceId: annualPrice.stripe_price_id,
     })
 
     if (emailIsValid(viewer?.email)) {
       await track('checkout: valid email present', {
-        priceId: annualPrice.price_id,
+        priceId: annualPrice.stripe_price_id,
       })
       await track('checkout: redirect to stripe', {
-        priceId: annualPrice.price_id,
+        priceId: annualPrice.stripe_price_id,
       })
-      stripeCheckoutRedirect(
-        annualPrice.price_id,
-        viewer.email,
-        account?.stripe_customer_id,
-        redirectURL,
+      stripeCheckoutRedirect({
+        priceId: annualPrice.stripe_price_id,
+        email: viewer.email,
+        stripeCustomerId: account?.stripe_customer_id,
         authToken,
-      )
+        quantity,
+      })
     } else {
       await track('checkout: get email', {
-        priceId: annualPrice.price_id,
+        priceId: annualPrice.stripe_price_id,
       })
-      setNeedsEmail(true)
+      router.push(
+        `/pricing/email?priceId=${annualPrice.stripe_price_id}&quantity=${quantity}`,
+      )
     }
   }
   return (
     <>
-      <div className="">
-        {!needsEmail && (
-          <div className="pt-12 sm:pt-16 lg:pt-20 dark:text-gray-100">
-            <div className="max-w-screen-xl mx-auto px-4 sm:px-6 lg:px-8">
-              <div className="text-center">
-                <h2 className="text-3xl leading-9 font-extrabold  dark:text-gray-100 text-gray-900 sm:text-4xl sm:leading-10 lg:text-5xl lg:leading-none">
-                  Your egghead membership pays for itself.
-                </h2>
-                <h3 className="text-2xl leading-9 font-extrabold dark:text-gray-100 text-gray-900 sm:text-3xl sm:leading-10 lg:text-4xl lg:leading-none">
-                  Over and over again.
-                </h3>
-                <p className="max-w-3xl m-auto mt-4 text-center text-xl leading-7 dark:text-gray-300 text-gray-600">
-                  Build your professional developer portfolio and learn the
-                  skills you need to climb the career ladder as a badass web
-                  developer
-                </p>
-              </div>
-            </div>
+      <div className="dark:bg-gray-900 bg-gray-50 dark:text-white text-gray-900 px-5">
```

Comment:
I deleted the "page" and moved it here instead of referencing a `PricingPage` component because that's what this is.